MapboxDirections/MBRouteLeg.swift

```diff
@@ -173,6 +182,15 @@ open class RouteLeg: NSObject, NSSecureCoding {
      */
     public let segmentCongestionLevels: [CongestionLevel]?
 
+    /**
+     An array containing the maximum speed along each road segment in the route leg geometry.
```

Comment:
Distinguish between maximum actual or expected speed and maximum posted speed limit:

```suggestion
     An array containing the maximum posted speed limit along each road segment in the route leg geometry.
```